Index: Services/UsersService.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Services/UsersService.cs b/Services/MedewerkersService.cs
rename from Services/UsersService.cs
rename to Services/MedewerkersService.cs
--- a/Services/UsersService.cs	(revision ed79f6b2c72265ea43fb6cfdfe8a1104821ed2c2)
+++ b/Services/MedewerkersService.cs	(date 1779787608572)
@@ -5,10 +5,10 @@
 
 namespace Chapeau.Services
 {
-    public class UsersService: IUsersService
+    public class MedewerkersService: IMedewerkersService
     {
         private readonly IMedewerkerRepository _medewerkerRepository;
-        public UsersService(IMedewerkerRepository medewerkerRepository)
+        public MedewerkersService(IMedewerkerRepository medewerkerRepository)
         {
             _medewerkerRepository = medewerkerRepository;
         }
Index: Services/IUsersService.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Services/IUsersService.cs b/Services/IMedewerkersService.cs
rename from Services/IUsersService.cs
rename to Services/IMedewerkersService.cs
--- a/Services/IUsersService.cs	(revision ed79f6b2c72265ea43fb6cfdfe8a1104821ed2c2)
+++ b/Services/IMedewerkersService.cs	(date 1779787608573)
@@ -2,7 +2,7 @@
 
 namespace Chapeau.Services
 {
-    public interface IUsersService
+    public interface IMedewerkersService
     {
         Medewerker? GetByLoginCredentials(string gebruikersnaam, string wachtwoord);
         public void Add(Medewerker medewerker);
Index: Controllers/TableController.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Controllers/TableController.cs b/Controllers/TafelController.cs
rename from Controllers/TableController.cs
rename to Controllers/TafelController.cs
--- a/Controllers/TableController.cs	(revision ed79f6b2c72265ea43fb6cfdfe8a1104821ed2c2)
+++ b/Controllers/TafelController.cs	(date 1779787608575)
@@ -7,17 +7,17 @@
 namespace Chapeau.Controllers
 {
     [Authorize]
-    public class TableController : Controller
+    public class TafelController : Controller
     {
-        private readonly ITablesRepository _tablesRepository;
-        public TableController(ITablesRepository tablesRepository)
+        private readonly ITafelRepository _TafelsRepository;
+        public TafelController(ITafelRepository TafelsRepository)
         {
-            _tablesRepository = tablesRepository;
+            _TafelsRepository = TafelsRepository;
         }
         public IActionResult Index()
         {
-            List<Table> tables = _tablesRepository.GetAll();
-            return View(tables);
+            List<Tafel> Tafels = _TafelsRepository.GetAll();
+            return View(Tafels);
         }
 
         /*public IActionResult Edit()
@@ -30,22 +30,22 @@
             {
                 return NotFound();
             }
-            Table? table = _tablesRepository.GetById((int)id);
-            return View(table);
+            Tafel? Tafel = _TafelsRepository.GetById((int)id);
+            return View(Tafel);
         }
         [HttpPost]
-        public ActionResult Edit(Table table)
+        public ActionResult Edit(Tafel Tafel)
         {
             try
             {
                 //update user via repository
-                _tablesRepository.Update(table);
+                _TafelsRepository.Update(Tafel);
                 //go back to users list (via index)
                 return RedirectToAction("Index");
             }
             catch (Exception ex)
             {
-                return View(table);
+                return View(Tafel);
             }
         }
     }
